reports/historical/_gmmp_2015_importer.py

```diff
@@ -304,6 +317,18 @@ def import_6(self, sheet_data):
             row_heading_col=2,
         )
 
+        for year, country_data_per_media_per_region in all_data.items():
+            if year != 2010:
```

Comment:
```suggestion
            if year > 2010:
```
One can hope in `2025` they'll be reusing our code.